src/main/assembly/dist/bag/metadata/amd/v0.1/amd.xsd

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2012 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified"
+    targetNamespace="http://easy.dans.knaw.nl/easy/dataset-administrative-metadata/"
+    xmlns:damd="http://easy.dans.knaw.nl/easy/dataset-administrative-metadata/" xmlns:wfs="http://easy.dans.knaw.nl/easy/workflow/">
+    <xs:import namespace="http://easy.dans.knaw.nl/easy/workflow/" schemaLocation="wfs.xsd"/>
+    <!-- import the versioned schema of wfs -->
+    <xs:annotation>
+        <xs:documentation>Schema to describe the administrative metadata for datasets in EASY, version 0.1.</xs:documentation>
```

Comment:
I assume this is the version that is currently in use in Fedora? Maybe mention that in this documentation, such that we still remember its history in 10 years...
Same for WFS.xsd - v0.1

Reply:
It will be documented in the DANS bagit profile, v0